src/compose/ComposeBox.js

```diff
@@ -90,7 +91,7 @@ type Props = $ReadOnly<{|
 type State = {|
   isMessageFocused: boolean,
   isTopicFocused: boolean,
-
+  isUploading: boolean,
   /** Almost the same as isMessageFocused || isTopicFocused ... except
```

Comment:
nit: leave the blank line here -- it's there intentionally, to separate the doc comment from the thing above it. That helps keep the comment visually attached to the thing it's about, which comes just after it.

Reply:
fixed.